src/scrapers/platformen/nlvoorelkaar.py

```diff
@@ -68,7 +68,11 @@ def complete(self, initiative: InitiativeImport):
 
             if self.config.group == InitiativeGroup.DEMAND:
                 title = soup.find("h2", "result__title")
-                initiative.organiser = title.contents[0]
```

Comment:
Hmmm, eigenlijk is de organiser altijd de 'Geplaatst door:' tenzij er dus ook een aangesloten bij is. Dit was de in bestaande code misschien ook niet goed. Je zou de code voor 'Aangesloten bij:' dan ook kunnen hergebruiken om de "geplaats door:" te vinden aangezien dat eenzelfde opmaak is.